eos/parameters/hadronic-matrix-elements.yaml

```diff
@@ -558,281 +630,336 @@ groups:
+          latex:    '$\alpha_{A_0,0}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^A0_1@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{A_0,1}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^A0_2@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{A_0,2}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^A1_0@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{A_1,0}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^A1_1@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{A_1,1}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^A1_2@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{A_1,2}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^A12_1@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{A_{12},1}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^A12_2@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{A_{12},2}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^V_0@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{V,0}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^V_1@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{V,1}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^V_2@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{V,2}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^T1_0@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{T_1,0}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^T1_1@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{T_1,1}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^T1_2@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{T_1,2}^{B_s \to K^*,\text{BSZ2015}}$'
           comments: ''
       'B_s->K^*::alpha^T2_1@BSZ2015' :
           central:  0.0
           min:      0.0
           max:      0.0
+          latex:    '$\alpha_{T_{2},1}^{B_s \to K^*,\text{BSZ2015}}$'
```

Comment:
Here again I would remove the unnecessary brackets for consistency with the rest